### PR TITLE
Define an alternative set of bindings (CTRL-G KEY)

### LINKED ISSUE #3: Ctrl-g Ctrl-b keybinding does not work if tmux prefix is Ctrl-b
This happens either in zsh and bash, not sure how to work around it other than using a different keybinding.

### LINKED ISSUE #4: Ctrl-g Ctrl-s keybinding does not work if flow control is active
This is expected, but since that is the default keybinding to open `_fzf_git_stashes`, it'll likely cause a bad user experience since the screen freeze if flow control is active, until the user hits <kbd>Ctrl-q</kbd>.

Maybe consider a different keybinding or tell people to disable flow control in the README?

### PR DESCRIPTION
**Pull request by junegunn**
@phelipetls What do you think?

Close #3
Close #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ Usage
     * <kbd>CTRL-G</kbd><kbd>CTRL-R</kbd> for **R**emotes
     * <kbd>CTRL-G</kbd><kbd>CTRL-H</kbd> for commit **H**ashes
     * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> for **S**tashes
+  > :warning: You may have issues with these bindings in the following cases:
+  >
+  > * <kbd>CTRL-G</kbd><kbd>CTRL-B</kbd> will not work if
+  >   <kbd>CTRL-B</kbd> is used as the tmux prefix (bindkey)
```
Comment (phelipetls) [inline README.md:34]: I didn't understand the "(bindkey)" at the end, does that refer to tmux or zsh?

File: README.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ Usage
...
+  > * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> will not work if flow control via
+  >   <kbd>CTRL-S</kbd> is enabled
```
Comment (phelipetls) [inline README.md:36]: I'd prefer to mention that the terminal might get "frozen" upon hitting <kbd>CTRL-S</kbd>. Something like that maybe?

```suggestion
  > * <kbd>CTRL-G</kbd><kbd>CTRL-S</kbd> will not work if flow control is enabled,
  >   <kbd>CTRL-S</kbd> will freeze the terminal instead
```

File: README.md [hunk 3]
```diff
@@ -28,6 +28,17 @@ Usage
...
+  >     * (`stty -ixoff -ixon` will disable it)
+  >
+  > To workaround the problems, you can use
+  > <kbd>CTRL-G</kbd><kbd>*{key}*</kbd> instead of
+  > <kbd>CTRL-G</kbd><kbd>CTRL-*{KEY}*</kbd>.
```
Comment (phelipetls) [inline README.md:41]: That's very smart.

**Review by phelipetls — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (junegunn) [reply README.md:34]: Oh, I'm gonna remove it.

File: README.md (hunk 2 above)
Comment (junegunn) [reply README.md:36]: Good idea.

File: README.md (hunk 1 above)
Comment (junegunn) [reply README.md:34]:
```suggestion
  >   <kbd>CTRL-B</kbd> is used as the tmux prefix
```

**Comment by junegunn** [comment]
Thanks for the review.

**Pull request merged**